Changelog.md

```diff
@@ -1,3 +1,19 @@
+## 4.0.0-beta7 - UNRELEASED
+
+## Changed
+
+-   Removed recording duration limitations by using the disk to keep the
+    collected data instead of RAM
+-   Export format is now '.ppk2'
+-   '.ppk' is deprecated and importing this format will auto generate the
+    '.ppk2' next to that same file
+-   Allow exporting of data even when data was loaded from file (hence allowing
+    partial export to .csv)
```

Comment:
```suggestion
-   Changed export format to `.ppk2`.  `.ppk` has been deprecated and importing this format will autogenerate a
    `.ppk2` file next to the `.ppk` file.
-   Exporting of data now works also when the data was loaded from file. This allows
    partial export to `.csv`.
-   Removed limitations for recording duration by using the disk to store the
    collected data instead of using RAM.
```
Why is it partial export to CSV?

Reply:
Because you either re export the same file (not much use for that) or a selection and at the moment selection can only export to CSV